### DISCUSSION TITLE
Install x86_64 app on an m1 Mac

### DISCUSSION THREAD

**Discussion by Tarang** [Homebrew/discussions#1007] [Getting started]
Hi,

I would like to force install an x86 version of a package.

If I do ` arch -x86_64 brew install ffmpeg` I get an error:

```
Error: Cannot install under Rosetta 2 in ARM default prefix (/opt/homebrew)!
To rerun under ARM use:
    arch -arm64 brew install ...
To install under x86_64, install Homebrew into /usr/local.
```

I would specifically like x86 version of this package so I don't want the arm version. How can I go about doing this?

**Comment by SMillerDev** [comment]
Run the brew installer again with Rosetta and use that install to install ffmpeg.

**Reply by Tarang** [reply]
But then won't that uninstall my m1 version of homebrew?

**Reply by SMillerDev** [reply]
No, there is a special uninstall script for that.

**Reply by Tarang** [reply]
I'm quite confused by this, because I installed both, both have installed at /opt/homebrew/bin/brew and if I use the rosetta terminal I am going to get a different outcome to the apple silicon terminal even though the installation is done twice?

Just confirming because it sounds right but it is a bit odd that both are installed in the same place

**Reply by gromgit** [reply]
> I installed both, both have installed at /opt/homebrew/bin/brew

If you used the standard installer, you just installed the M1 Homebrew version _twice_ in the same place.

To install the Intel Homebrew hierarchy, I believe the following should work:
```
arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```
That should tell you that you're installing under `/usr/local`, not `/opt/homebrew`.
